admin/tools/qserv-install.sh

```diff
@@ -7,7 +7,7 @@ DIR=$(cd "$(dirname "$0")"; pwd -P)
 
 # Default values below may be overidden by cmd-line options
 MODE="internet mode"
-NEWINSTALL_URL="https://sw.lsstcorp.org/eupspkg/newinstall.sh"
+NEWINSTALL_URL="https://raw.githubusercontent.com/lsst/lsst/12.0/scripts/newinstall.sh"
```

Comment:
Will the new one always be at this location or will it require constant updating? I wonder why the "12.0" is there.